### PR TITLE
New role variable for setting a group name different from solr_user

### PR DESCRIPTION
**Pull request by diadzine**
Add solr_group role variable to allow user to set different group name from solr_user.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,9 @@ Files will be downloaded to this path on the remote server before being moved in
 
     solr_create_user: true
     solr_user: solr
+    solr_group: solr
```
Comment (geerlingguy) [inline README.md:23]: Can you update this default to be `solr_group: "{{ solr_user }}"` so it's more obvious that that's the default?

**Review by geerlingguy — CHANGES_REQUESTED** [review]
Please see inline comment - once that's done I'll merge, thanks!

File: README.md (hunk 1 above)
Comment (diadzine) [reply README.md:23]: Sure!

Comment (diadzine) [reply README.md:23]: I followed the same model from the README.md ```solr_install_path: /opt/solr```
which is ```solr_install_path: "/opt/{{ solr_service_name }}"``` in defaults/mail.yml

**Review by geerlingguy — APPROVED** [review]

**Pull request merged**